blogposts/gitlab-adds-code-navigation-and-code-intelligence-by-sourcegraph.md

```diff
@@ -0,0 +1,90 @@
+---
+title: GitLab adds code navigation and code intelligence by Sourcegraph
+author: Christina Forney
+publishDate: 2019-11-12T10:00-07:00
+tags: [
+  blog
+]
+slug: gitlab-adds-code-navigation-and-code-intelligence-by-sourcegraph
+heroImage: https://about.sourcegraph.com/blog/gitlab-integration-symbols-preview-dark.jpg
+published: true
+---
+<p style="text-align: center">
+  <img src="images/gitlab-integration-banner-dark.jpg" />
+</p>
+
+**"Every single developer that used Sourcegraph and talked to me about it loved the product. Every code product must have intelligent navigation."** - [Sid Sijbrandij, GitLab CEO](https://news.ycombinator.com/item?id=18118924)
+<br>
+<br>
+
+Over 100,000 organizations use GitLab to manage their DevOps lifecycle. Today GitLab announced a collaboration with Sourcegraph to make code navigation and code intelligence available to all GitLab users, integrated seamlessly in the GitLab developer workflow and user interface.
```

Comment:
Can we update this to link to their blog post when it goes live?

Reply:
Ok!